lib/constants.js

```diff
@@ -1,34 +1,34 @@
 const { hex } = require('./helpers');
 
 const StakingUintParamType = {
-  MIN_STAKE: hex('MIN_STAK'),
-  MAX_EXPOSURE: hex('MAX_EXPO'),
-  MIN_UNSTAKE: hex('MIN_UNST'),
-  UNSTAKE_LOCK_TIME: hex('UNST_LKT'),
+  MIN_STAKE: hex('MIN_STAK'.padEnd(8, '\0')),
+  MAX_EXPOSURE: hex('MAX_EXPO'.padEnd(8, '\0')),
+  MIN_UNSTAKE: hex('MIN_UNST'.padEnd(8, '\0')),
+  UNSTAKE_LOCK_TIME: hex('UNST_LKT'.padEnd(8, '\0')),
 };
 
 const PoolUintParamType = {
-  minPoolEth: hex('MIN_ETH'),
+  minPoolEth: hex('MIN_ETH'.padEnd(8, '\0')),
 };
 
 const PoolAddressParamType = {
-  swapOperator: hex('SWP_OP'),
-  priceFeedOracle: hex('PRC_FEED'),
+  swapOperator: hex('SWP_OP'.padEnd(8, '\0')),
```

Comment:
This was reverting without the 0 padding. I wasn't sure if the other ones were ok, so I added it there also.